sensor/common/metrics/metrics.go

```diff
@@ -148,9 +150,11 @@ var (
 	})
 
 	telemetryLabels = prometheus.Labels{
-		"sensor_version": version.GetMainVersion(),
+		"branding":       branding.GetProductNameShort(),
+		"build":          buildinfo.BuildFlavor,
```

Comment:
We could use here https://github.com/stackrox/stackrox/blob/c5681c1db28266d56a8e79465dc1fddd757d87ab/pkg/version/version.go#L189
but this is fine too. We just need to remember to filter release builds with `-` in version later in the Grafana

Reply:
Used this function and changed the possible values of `build` to `release` or `internal` (includes dev and test builds).